<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Import Data</class>
 <widget class="QMainWindow" name="Import Data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>비식별 처리 기법</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>861</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>비식별기법</string>
    </property>
    <widget class="QRadioButton" name="Method6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>370</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>라운딩</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Method5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>320</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>총계 처리</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Method2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>재배열</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Method1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>교환 처리</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Method4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>마스킹 및 삭제</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Method3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>범주화</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>공통</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>수치데이터</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>861</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>539</x>
      <y>570</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>다음 &gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>취소</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>851</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>비식별 처리 기법</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
